Tests/TestingApps/PurchaseTesterSwiftUI/Core/ConfiguredPurchases.swift

```diff
@@ -49,7 +49,9 @@ public final class ConfiguredPurchases {
                 .with(observerMode: observerMode)
                 .with(entitlementVerificationMode: entitlementVerificationMode)
                 #if DEBUG
-                .with(dangerousSettings: .init(autoSyncPurchases: true, internalSettings: DangerousSettings.Internal(usesStoreKit2JWS: useStoreKit2)))
+                .with(dangerousSettings: .init(autoSyncPurchases: true,
+                                               internalSettings: DangerousSettings.Internal(usesStoreKit2JWS: useStoreKit2),
+                                               disallowSharingAppStoreAccountsForAnonymousIDs: true))
```

Comment:
@MarkVillacampa there isn't a way to force SK2 JWT before iOS v5, right? If so I'll have to rebase this PR on that

Reply:
Not an easy way to force SK2 JWT before v5, no. But do we *have* to check that flag to use SK2 for this very specific dangerous setting? I'd assume if somebody enables it they know what they're doing :D

Reply:
But I mean, you _can't_ use the flag in any case because it's internal, so they wouldn't be able to access it unless they do @testable (what this sample app is doing) or otherwise fork the SDK, right?